test/modules/deitz/test_module_class_conflict2.good

```diff
@@ -1,2 +1 @@
 test_module_class_conflict2.chpl:13: error: 'use' of non-module/enum symbol MC
-test_module_class_conflict2.chpl:2: error: Definition of symbol MC
```

Comment:
I don't think this is the most important error message or anything but why is it going away?

Reply:
It's because of the change to only return the name of a symbol in lookupForImport if it isn't a module (or an enum for use statements).  I could restore that line number information, but it would mean going back to the implementation where we drop the symbol on the floor and only keep the name after performing that check

Reply:
It seems like an acceptable tradeoff for now if the compiler code is simpler.